strawberry_django_plus/permissions.py

```diff
@@ -227,7 +232,7 @@ def get_with_perms(pk, info, *, required=False, model=None):
     user = cast(StrawberryDjangoContext, info.context).request.user
     for check in checks:
         f = any if check.any else all
-        checker = check.obj_perm_checker(info, cast(UserType, user))
+        checker = check.obj_perm_checker(info, cast(AbstractBaseUser, user))
```

Comment:
With this change you are excluding anonymous user from the type, since it doesn't inherit from `AbstractBaseUser`

Reply:
It actually does

Reply:
oh, you are right. I fix this

Reply:
optimal would be a Protocol provided by django for typing